modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApi.mustache

```diff
@@ -15,23 +15,25 @@ import javax.validation.constraints.*;
 {{#operations}}
 public class {{classname}}ControllerImp {{#useInterfaces}}implements {{classname}}ControllerImpInterface{{/useInterfaces}} {
 
+    {{#examples}}
```

Comment:
I don't know neither. I never used the example label, but with the last commit we were generating code that was not compiling because of this label.